<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <widget class="QLineEdit" name="line_name">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_email">
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_capture">
     <property name="text">
      <string>Capture Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_submit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </layout>
  </widget>
 </widget>
</ui>
